Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -1132,6 +1158,19 @@ module.exports = {
       .setFunctionName('getLinearVelocityLength')
       .setManipulatedType('number');
 
+	aut
+      .addExpression(
+        'LinearVelocityLength',
+        t('Linear velocity Length'),
+        t('Get an objects linear velocity length.'),
+        t('Velocity'),
+        'res/physics16.png'
+      )
+      .addParameter('object', t('Object'), '', false)
+      .addParameter('behavior', t('Behavior'), 'Physics2Behavior')
+      .getCodeExtraInformation()
+      .setFunctionName('getLinearVelocityLength');  
```

Comment:
@Lizard-13 Should that be just called `getLinearVelocity`? It seems strange to me to have Length associated to velocity, which already convey the meaning of the unit. You would not say "speed length" right? I know that velocity is to be understand like a vector, but still sounds strange to me.

In which case the expression would be just named LinearVelocity

Reply:
For the final user it should be `getLinearVelocity`, yes. But as you say, in the code velocity is a vector, so it would be weird too that `getLinearVelocity` doesn't return the vector.

I can change it if you want, none of the options really bother me :)

Reply:
Let's keep getLinearVelocityLength for the function name then, and LinearVelocity for the expression.